main/src/library/File.flix

```diff
@@ -119,6 +119,51 @@ namespace File {
                 Err(getMessage(ex))
         }
 
+    ///
+    /// Returns a string containing the given file `f`.
+    ///
+    pub def read(f: String): Result[String, String] & Impure =
+        try {
+            import new java.io.File(String) as newFile;
+            import java.io.File.toPath();
+            import java.nio.file.Files:readString(##java.nio.file.Path);
+            let javaFile = newFile(f);
+            let javaPath = toPath(javaFile);
+            let result = readString(javaPath);
+            Ok(result)
+        } catch {
+            case ex: ##java.io.IOException =>
+                import java.lang.Throwable.getMessage();
+                Err(getMessage(ex))
+        }
+
+    ///
+    /// Returns a string containing the given file `f` with the options `opts`.
+    /// TODO: add a note on what exactly the opts do
```

Comment:
I don't know how to explain the opts properly

Reply:
The [`FileInputStream.read`](https://docs.oracle.com/javase/7/docs/api/java/io/FileInputStream.html#read(byte[],%20int,%20int)) Javadoc may serve as inspiration:

```
    off - the start offset in the destination array b   /// our `offset`
    len - the maximum number of bytes read.             /// our `count`
```

Then there's the [`String` constructor](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#String(byte[],%20java.nio.charset.Charset)) Javadoc:

```
    charset - The charset to be used to decode the bytes
```